.github/workflows/publish-workflows-service.yml

```diff
@@ -220,7 +220,8 @@ jobs:
           changes: |
             {
               "kubernetes/helm/wf-service/${{steps.update_helm_check.outputs.file_name}}": {
-                "image.tag": "${{ needs.build-and-push-ee-image.outputs.docker_tag }}"
+                "image.tag": "${{ needs.build-and-push-ee-image.outputs.docker_tag }}",
+                "prismaMigrate.image.tag": "${{ needs.build-and-push-image.outputs.docker_tag }}"
```

Comment:
prisma migrate should be the same image as our current deployment
and what about data-migration?

Reply:
The current deployment in wf-service uses ee image. In the requirement analysis we discussed that the prismaMigrate will use image without prefix ee.
Regarding the data-migration and db-sync, since both of these jobs use image with prefix ee it will be pulled from the "image.tag"